redis/manifests/redis.yaml

```diff
@@ -282,42 +282,42 @@ spec:
       protocol: TCP
       targetPort: metrics
   selector:
-    app.kubernetes.io/instance: redis-shared
-    app.kubernetes.io/name: redis-shared
+    app.kubernetes.io/instance: redis
+    app.kubernetes.io/name: redis
 ---
 # Source: redis/templates/master/application.yaml
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: redis-shared-master
-  namespace: "dev-enablement"
+  name: redis-master
+  namespace: "default"
   labels:
-    app.kubernetes.io/instance: redis-shared
+    app.kubernetes.io/instance: redis
     app.kubernetes.io/managed-by: Helm
-    app.kubernetes.io/name: redis-shared
+    app.kubernetes.io/name: redis
     app.kubernetes.io/version: 7.2.2
     helm.sh/chart: redis-18.1.6
     app.kubernetes.io/component: master
   annotations:
-    app.uw.systems/description: cache for opslevel-k8s-deployer
+    app.uw.systems/description: change-me
```

Comment:
:thinking: should we just exclude this instead?

* Include it in the example(s)
* If teams forget it they get an :x: on OpsLevel which is possibly better than forgetting to set this and having a placeholder value there

Reply:
[Good idea. Removed default values](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/68d08076192f871ea58d454ee525adc6899e4284#diff-4bf6eab5a75bca36be6a8d06c69efa33c91d1cca432f829251c57cea5d59daefL11)